apps/schema/static/schema/swagger.yaml

```
@@ -176,6 +176,56 @@ paths:
               allOf:
               - $ref: '#/components/requestBodies/shipment/createAttributes'
 
+  /api/v1/shipments/{shipment_id}/tracking/:
+    post:
+      summary: Add tracking data
+      description: >
+        Get additional info about a `Shipment`.
```

Please be careful of copy/pasting descriptions from other endpoints.
```suggestion
        Associate GPS tracking data with a `Shipment`.
```